test/core/tsi/ssl_transport_security_test.cc

```diff
@@ -1244,13 +1244,15 @@ TEST(SslTransportSecurityTest, MainTest) {
       // BoringSSL and OpenSSL have different behaviors on mismatched ALPN.
       ssl_tsi_test_do_handshake_alpn_client_no_server();
       ssl_tsi_test_do_handshake_alpn_client_server_mismatch();
-#endif
-      ssl_tsi_test_do_handshake_alpn_server_no_client();
-      ssl_tsi_test_do_handshake_alpn_client_server_ok();
+      // These tests fail with openssl3 and openssl111 currently but not
```

Comment:
I haven't investigated why they fail, I just verified locally that they also fail when building with OpenSSL 1.1.1, so them failing with OpenSSL3 is not unexpected. They've just not been tested against OpenSSL recently, so I guess what I'm saying is at the least this isn't a regression from the current state.

I'm also looking into why they fail

Reply:
OK gotcha. If it's a build error that also exists for OpenSSL 1.1.1, then that's less concerning. It would be good to get to the bottom of the failure, if possible. Please feel free to send me any logs if I can be helpful.